### ISSUE TITLE
inconsistent space handling in inline plugin templates when switching in and out of editing mode

### ISSUE THREAD

**Issue by lddubeau** [divio/django-cms#4358]
### Versions

Django CMS: 3.1.2, 3.1.0
Django: 1.8, 1.7

### Steps to reproduce

1. Create a project in which you have this plugin:

    ```
    from cms.models.pluginmodel import CMSPlugin
    from cms.plugin_base import CMSPluginBase
    from django.utils.translation import ugettext as _
    from cms.plugin_pool import plugin_pool

    class Plugin(CMSPluginBase):
        text_enabled = True
        model = CMSPlugin
        name = _("Something")
        render_template = "something/something.html"
        cache = False

        def render(self, context, instance, placeholder):
            return context

    plugin_pool.register_plugin(Plugin)
    ```

  The `something/something.html` file contains:

  ```
  <div>
    <span>Foo</span> <span>Bar</span>
  </div>
  ```

  Edit the above code as needed to make it work in your project.

2. Instantiate a Text plugin somewhere and instantiate a Something plugin in it. Save the Text plugin and switch back to "Content" view.

3. Click "Save as draft" and then click "Publish".

### Expected behavior

The text of the Something instance should always be "Foo Bar" with a space between the two words.

### Actual behavior

When in editing mode (step 2 above), the text appears as "FooBar". When out of editing mode, it appears as "Foo Bar" (step 3 above).

### Observations

Adding any character other than a space between the spans that contain "Foo" and "Bar" prevents the problematic behavior from happening. For instance if I put a period just after the `</span>` that appears after "Foo", the rendered plugin is consistently: "Foo. Bar" whether in edit mode or not. It looks like something similar to the `{% spaceless %}` tag is stripping spaces when in edit mode.

I've searched the documentation but found nothing about this behavior.

Ultimately, once published the rendering is correct, but the incorrect rendering in editing mode could lead staff members to file spurious bug reports to the developers of a site that uses Django CMS.

**Comment by FinalAngel** [comment]
This is related to https://github.com/divio/django-cms/pull/4458

**Comment by yakky** [comment]
Fixed in #4467
